charms/kfp-viz/src/components/pebble_components.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from charmed_kubeflow_chisme.components.pebble_component import PebbleServiceComponent
+from ops.pebble import Layer
+
+logger = logging.getLogger(__name__)
+
+
+class KfpVizPebbleService(PebbleServiceComponent):
+    def get_layer(self) -> Layer:
+        """Pebble configuration layer for ml-pipeline-visualizationserver."""
+        layer = Layer(
+            {
+                "services": {
+                    self.service_name: {
+                        "override": "replace",
+                        "summary": "entry point for ml-pipeline-visualizationserver",
+                        "command": "python3 server.py",  # Must be a string
```

Comment:
In the ROCK command for this service is `python3.6 /server.py`. Can we ensure that were running the same version of Python.

Reply:
That or you could try using the [rock](https://hub.docker.com/r/charmedkubeflow/visualization-server), which already sets the command for you. Similar to what was done in `kfp-schedwf`.